recursive_art.py

```diff
@@ -20,9 +26,25 @@ def build_random_function(min_depth, max_depth):
         (See the assignment writ-eup for details on the representation of
         these functions)
     """
-    # TODO: implement this
-    pass
-
+    building_blocks = ['x','y','cos_pi','sin_pi','prod','avg','tan_pi','substract',]
+    pick = random.choice(building_blocks)
+    i = random.randint(min_depth, max_depth)
```

Comment:
something like `random_depth` would be a more appropriate variable name. `i` does not give you any semantic information about the variable.

Reply:
but I don't see you using it anywhere else in the code.